chapter_optimization/lr-scheduler.md

```diff
@@ -356,15 +441,14 @@ In some cases initializing the parameters is not sufficient to guarantee a good
 A rather simple fix for this dilemma is to use a warmup period during which the learning rate *increases* to its initial maximum and to cool down the rate until the end of the optimization process. For simplicity one typically uses a linear increase for this purpose. This leads to a schedule of the form indicated below.
 
 ```{.python .input}
-scheduler = lr_scheduler.CosineScheduler(20, warmup_steps=5, base_lr=0.5,
+scheduler = lr_scheduler.CosineScheduler(20, warmup_steps=5, base_lr=0.3,
```

Comment:
why do we change base_lr?

Reply:
PyTorch implementation was not converging regularly somehow when base_lr was 0.5. So I changed it across frameworks for consistency.

Reply:
That's ok. Just give me a heads up if you change original mx code :)

Reply:
Sure! I mentioned it [here](https://github.com/d2l-ai/d2l-en/pull/1305#discussion_r467208799). Will make it more explicit in the future if mxnet code is changed.